Tests/PostgresNIOTests/New/PSQLChannelHandlerTests.swift

```diff
@@ -32,7 +32,7 @@ class PSQLChannelHandlerTests: XCTestCase {
     
     func testEstablishSSLCallbackIsCalledIfSSLIsSupported() {
         var config = self.testConnectionConfiguration()
-        config.tlsConfiguration = .forClient(certificateVerification: .none)
+        config.tlsConfiguration = .makeClientConfiguration()
```

Comment:
We seem to not be passing the certificate verification in the new version?

Reply:
In the case of the unit test, we don't care at all.